www/source/partials/docs/_accept-the-license.html.md.erb

```diff
@@ -0,0 +1,9 @@
+## <a name="accept-the-license" id="accept-the-license" data-magellan-target="accept-the-license">Accept the License</a>
+Before you can use Habitat, you must accept the license. There are a variety of ways to accept the license - which one you choose will depend on your use case.
+
+* You can do nothing, and the first time you run any <code>hab</code> command, you will get an interactive prompt that will ask you to accept the license. Assuming you do, a file will get written to disk and you won't be prompted again.
+* You can run <code>hab license accept</code> and it will accept the license and write the necessary metadata file to disk without prompting you.
+* You can create an empty file at either <code>/hab/accepted-licenses/habitat</code> (if your hab commands run as root) or <code>$HOME/.hab/accepted-licenses/habitat</code> (if they run as a user other than root). Note that if you have Habitat commands running as a mix of users, creating both files is a good idea.
+* You can export the <code>HAB_LICENSE</code> environment variable with either of two values:
+  * <code>accept</code> will accept the license and write the necessary metadata to disk, avoiding any future prompts. This behavior is identical to running <code>hab license accept</code>.
+  * <code>accept-no-persist</code> will accept the license for 1 command invocation only - no metadata will get written to disk and you will be prompted again in the future. This is most suitable for CI systems or other areas where persisting files to disk is undesireable.
```

Comment:
instead of `for 1 command invocation only`, perhaps this should be `within the scope of the environment variable.`

Reply:
Sure